substrate/frame/support/procedural/src/lib.rs

```diff
@@ -972,6 +972,15 @@ pub fn event(_: TokenStream, _: TokenStream) -> TokenStream {
 	pallet_macro_stub()
 }
 
+///
+/// ---
+///
+/// Documentation for this macro can be found at `frame_support::pallet_macros::include_metadata`.
```

Comment:
I'm actually not sure it is needed. I think we can just have the doc here and reexport in frame_support with `[doc(inline)]`.

Reply:
Hmmm, I've kept consistency here with `pallet::storarge` / call / call index a bit down below 🤔

Reply:
Yes ok. I think all of them should be written here directly. That would help user using rust-analyzer. But we can do in a follow-up.